homeassistant/util/unit_system.py

```diff
@@ -291,6 +293,8 @@ def _deprecated_unit_system(value: str) -> str:
         ("gas", UnitOfVolume.CUBIC_METERS): UnitOfVolume.CUBIC_FEET,
         # Convert non-USCS precipitation
         ("precipitation", UnitOfLength.MILLIMETERS): UnitOfLength.INCHES,
+        # Convert non-USCS pressure
+        ("pressure", UnitOfPressure.BAR): UnitOfPressure.PSI,
```

Comment:
We probably want to convert a few more units than just `bar` to `psi`. Maybe convert all except `inHg` and `psi` to `psi`?
Maybe consider converting `mmHg` to `inHg` since it's likely to be an atmospheric pressure.